src/ssl.c

```diff
@@ -17702,6 +17702,106 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
     }
     #endif /* !defined(HAVE_FIPS) || (defined(HAVE_FIPS_VERSION) && \
               (HAVE_FIPS_VERSION > 2)) */
+
+#if !defined(HAVE_FIPS) && !defined(HAVE_SELFTEST)
+#if !defined(WOLFSSL_NOSHA512_224)
+    int wolfSSL_SHA512_224_Init(WOLFSSL_SHA512_224_CTX* sha)
+    {
+        int ret;
+
+        typedef char 
+                sha_test[sizeof(SHA512_224_CTX) >= sizeof(wc_Sha512) ? 1 : -1];
+        (void)sizeof(sha_test);
+
+        WOLFSSL_ENTER("SHA512_224_Init");
```

Comment:
Any reason to not use the actual function name, `wolfSSL_SHA512_224_Init`? Same goes for other enters/leaves in this PR.

Reply:
No not special reason. Just followed the way of wolfSSL_Sha512 functions. Replaced those comments with their actual function name.